src/app/ubs-admin/components/shared/components/pagination/pagination.component.scss

```diff
@@ -0,0 +1,26 @@
+.custom-pagination {
+  display: flex;
+  justify-content: space-between;
+  width: 360px;
+
+  .arrow {
+    vertical-align: middle;
+    color: #64727d;
+  }
+
+  div {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border-radius: 4px;
+
+    .page {
+      padding: 4px 8px;
+      color: #444e55;
+    }
+  }
+
+  .current {
+    background-color: #13aa57;
```

Comment:
use var(--ubs-primary-green) instead of this.

Reply:
Done